core/src/api/controllers/plugins.controller.js

```diff
@@ -24,6 +23,22 @@ module.exports = {
             return res.status(500).send(error);
         })
 
+    },
+
+    delete: (req, res) => {
+        //get plugin config file
+        if (!req.body.key || environment.key !== req.body.key) {
+            return res.status(500).send("Invalid key");
```

Comment:
@DavidIzaac you should not check the key. it is automatically checked for any post request in `app.js`